### DISCUSSION TITLE
Yourls homepage is different

### DISCUSSION THREAD

**Discussion by lazycan** [YOURLS/YOURLS#3205] [Q&A] [answered]
Hi,
How to make the home page of Yourls look like this (see attached image). A colleague told me that he has this page directly at work
![homepage-yourls](https://user-images.githubusercontent.com/55274052/150133206-4afca806-44ab-4eea-8102-ae5ddb5c649d.png)
.

**Comment by ozh** [comment, marked as answer]
http://yourls.org/#FAQ;noindex
